AGENTS.md

```
@@ -1,110 +1,25 @@
-# Agent Verification Guide
+## Verification commands
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)